tests/jvm/src/test/resources/lowlevel.expect

```diff
@@ -588,6 +594,27 @@ _root_.types.Test.C#x. => val x: : C.this.p.X
   C => _root_.types.Test.C#
   p => _root_.types.Test.C#p.
   X => _root_.types.P#X#
+_root_.types.Test.Literal. => final object Literal
+_root_.types.Test.Literal.bool. => final val bool: : true
```

Comment:
@xeno-by do you know why there are two colons `: :`? I expected there to be only one colon, the type is
```scala
tag: METHOD_TYPE
methodType {
  result {
    tag: LITERAL_TYPE
    literalType {
      tag: ENUM
      primitive: 0
      string: "_root_.java.nio.file.LinkOption.NOFOLLOW_LINKS."
    }
  }
}
```

Reply:
`: T` corresponds to `NullaryMethodType`, which is the signature of the underlying getter.